### ISSUE TITLE
Image upload error - build 70839

### ISSUE THREAD

**Issue by orchardc** [OrchardCMS/OrchardCore#3275]
I found all image upload failed, regardless the size/type of the image.

![image](https://user-images.githubusercontent.com/16193516/53693851-0cc6e800-3d9e-11e9-8813-8f4b84a4ccaf.png)

**Comment by scleaver** [comment]
Duplication of #3263

**Comment by sebastienros** [comment]
Could someone give me a repro starting from a clean setup? I understand the issue but can't repro locally.

**Comment by jtkech** [comment]
Just tried, i could repro.

- Last dev, run from VS in debug mode, blog setup, upload an image, it works.

- Publish the app in a folder, run from this folder, upload an image.

![upload](https://user-images.githubusercontent.com/8586360/53759994-eac68600-3ec1-11e9-89f5-46a2fe6c088b.png)

Can't right now, but, because we can repro, i will dig into it asap.

**Comment by sebastienros** [comment]
Try to disable the AntiforgeryTokenFilterAttribute, that was the issue when I developped locally, but I did an change that was supposed to fix it. The order of the filters is very important, and the new one should be before AF (using its default Order property). If that's the case, try to use a different default Order for the MediaSize attribute. But it would show that the version we publish with and the one in dev have different values (maybe log these values to be sure that it's the case).
